lib/chf/import/credit_builder.rb

```diff
@@ -1,33 +0,0 @@
-module Chf
```

Comment:
This was related to the upgrade from sufia 6.x to sufia 7. The code it built on only exists in the sufia 7.2-migration branch, so it is both no longer needed and no longer working (the tests fail).